### PR TITLE
Nom module

### PR DESCRIPTION
**Pull request by kchin28**
pls look over injection setup :'(

### CODE REVIEW

File: dbot/dbot/Program.cs [hunk 1]
```diff
@@ -38,6 +39,9 @@ public async Task MainAsync()
 
         public async Task InstallCommands() {
             client.MessageReceived += HandleCommand;
+
+            serviceCollection.AddSingleton(new NominationsService());
```
Comment (Loonman) [inline dbot/dbot/Program.cs]: I think this needs to be above before calling buildserviceprovider

File: dbot/dbot/Services/NominationsService.cs [hunk 2]
```diff
@@ -1,5 +1,11 @@
 
 namespace dbot.Services
 {
-    
+    public class NominationsService
+    {
+
+        public bool addNom() {
+            return false;
```
Comment (Loonman) [inline dbot/dbot/Services/NominationsService.cs]: no nominations for you

File: dbot/dbot/Program.cs (hunk 1 above)
Comment (kchin28) [reply dbot/dbot/Program.cs]: aite

File: dbot/dbot/Services/NominationsService.cs (hunk 2 above)
Comment (kchin28) [reply dbot/dbot/Services/NominationsService.cs]: v sad

**Pull request merged**